sunpy/timeseries/sources/eve.py

```diff
@@ -175,7 +193,7 @@ class EVESpWxTimeSeries(GenericTimeSeries):
     @peek_show
     def peek(self, column=None, **kwargs):
         """
-        Plots the time series in a new figure. An example is shown below:
+        Plots the time series in a new figure.
 
         ..
```

Comment:
Can you remove this and see if it works now?

Reply:
The plot? It works right